dds_web/__init__.py

```diff
@@ -171,6 +171,9 @@ def create_app(testing=False, database_uri=None):
         # Initiate app object
         app = flask.Flask(__name__, instance_relative_config=False)
 
+        # All variables in the env that start with FLASK_* will be loaded into the app config
+        app.config.from_prefixed_env()
+
```

Comment:
Nice!! 

I think we should have a small note regarding that the prefix is dropped though so that it's not confusing in the future. 
```suggestion
        # All variables in the env that start with FLASK_* will be loaded into the app config
        # 'FLASK_' will be dropped, e.g. FLASK_TESTVAR will be loaded as TESTVAR
        app.config.from_prefixed_env()

```